src/sass/formio-pretty-checkbox.scss

```diff
@@ -1 +1,5 @@
 @import '~pretty-checkbox/src/pretty-checkbox.scss';
+@import './fontawesome/fontawesome.scss';
```

Comment:
These should not be included. If they want font-awesome, then they can include this as part of their application. It should not be up to this library to do this.